src/blocks/Media/__stories__/Media.mdx

```diff
@@ -29,4 +29,6 @@ import * as MediaStories from './Media.stories.tsx';
 
 [`buttons?: Button[]` — An array with button objects](?path=/docs/documentation-types--docs#button)
 
+`border?: boolean` — Image border
```

Comment:
Can we use here types border: 'line' | 'shadow' | 'none'

we can set 'shadow' as !disableShadow and deprecate this prop

Reply:
sure